### PR TITLE
EnvelopeSkeleton: make oneline envelopes actually one line and not two

### PR DESCRIPTION
**Pull request by GVodyanov**
Fix https://github.com/nextcloud/mail/issues/10043

| Before | After |
| - | - |
| image | Screenshot from 2024-08-23 14-46-35 |

### CODE REVIEW

File: src/components/EnvelopeSkeleton.vue [hunk 1]
```diff
@@ -552,24 +552,51 @@ export default {
...
 		}
 
 		a {
 			margin: 0;
+			align-items: center;
+			height: unset;
+		}
+
+		:deep(.avatardiv__initials) {
+			line-height: 40px !important;
```
Comment (ChristophWurst) [inline src/components/EnvelopeSkeleton.vue:594]: why the deep overwrite?

Comment (GVodyanov) [reply src/components/EnvelopeSkeleton.vue:594]: Double-checked and it is indeed not needed. The idea was to have the avatar be `40px` in oneline and `45px` (as per Marco's suggestion some time ago), but now I noticed that Greta made it `40px` everywhere and not just in oneline, I assume per a new design decision.

Comment (GretaD) [reply src/components/EnvelopeSkeleton.vue]: yes, the avatar should be 40px everywhere on mail :)

**Review by GretaD — APPROVED** [review]
tested and looks good

**Pull request merged**